### PR TITLE
HELP-24133: seperate the page and default profiles 

### PR DESCRIPTION
**Pull request by k-anderson**
This allows the entry-tones to be removed from page group profile.

### CODE REVIEW

File: applications/conference/src/conf_config_req.erl [hunk 1]
```diff
@@ -56,6 +58,13 @@ default_profile() ->
                     ,kz_json:from_list(?DEFAULT_PROFILE_CONFIG)
                     ).
 
+-spec page_profile() -> wh_json:object().
+page_profile() ->
+    whapps_config:get(?CONFIG_CAT
```
Comment (jamesaimonetti) [inline applications/conference/src/conf_config_req.erl]: Master uses kapps_config now

File: applications/conference/src/conf_config_req.erl [hunk 2]
```diff
@@ -56,6 +58,13 @@ default_profile() ->
...
+                      ,[<<"profiles">>, ?PAGE_PROFILE_NAME]
+                      ,wh_json:from_list(?PAGE_PROFILE_CONFIG)
```
Comment (jamesaimonetti) [inline applications/conference/src/conf_config_req.erl]: kz_json too

File: applications/conference/src/conf_config_req.erl [hunk 3]
```diff
@@ -85,7 +94,8 @@ max_members_sound(Conference) ->
     end.
 
 -spec get_conference(ne_binary()) -> kapps_conference:conference().
-get_conference(?DEFAULT_PROFILE_NAME) -> kapps_conference:new();
+get_conference(?DEFAULT_PROFILE_NAME) -> whapps_conference:new();
```
Comment (jamesaimonetti) [inline applications/conference/src/conf_config_req.erl]: kapps_conference

**Comment by jamesaimonetti** [comment]
`make xref` is your friend!

**Comment by jamesaimonetti** [comment]
`make apis` will also need to be run so you can get latest changes to swagger.json and `system_config.conferences` schema docs

**Comment by k-anderson** [comment]
@jamesaimonetti I think I pushed my 3.22 branch as the -master :(  I have all those changes locally, will sort out and run make apis

**Pull request closed without merging**